tests/transformers/test_transformers_model_export.py

```diff
@@ -2907,3 +2907,43 @@ def test_whisper_model_serve_and_score_with_timestamps(whisper_pipeline, raw_aud
             "text"
         ]
     )
+
+
+@pytest.mark.skipif(
+    Version(transformers.__version__) < Version("4.29.0"), reason="Feature does not exist"
+)
+@pytest.mark.skipif(RUNNING_IN_GITHUB_ACTIONS, reason="url dependency")
+@pytest.mark.skipcacheclean
+def test_whisper_model_with_url(whisper_pipeline):
+    artifact_path = "whisper_url"
+
+    url = "https://www.nasa.gov/mp3/590325main_ringtone_kennedy_WeChoose.mp3"
```

Comment:
Can we commit an audio file in this repo and use it?

Reply:
ah good point. I'll convert to pulling the .wav file I added in the last audio PR :)